notebooks/official/prediction/llm_streaming_prediction.ipynb

```diff
@@ -0,0 +1,791 @@
+{
```

Comment:
<ul><li>Why the models are not consistent between these two places, one is using "google/text-bison@001" and another is using "text-bison".</li><li>No need to import again "from vertexai.preview.language_models import TextGenerationModel"</li></ul><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
done